sdks/java/core/src/test/java/org/apache/beam/sdk/options/PipelineOptionsTest.java

```diff
@@ -86,6 +86,7 @@ public void testDynamicAs() {
   }
 
   @Test
+  @Ignore("Broken with removal of runners from core. Consider requiring new DirectRunner")
```

Comment:
I think it is safe to delete tests that are broken. This test can live in the direct runner.